documentation/QUICKSTART.md

```diff
@@ -2,135 +2,116 @@
+- `BASE_DIR` - Set this to the directory where you want to store your outputs and datasets. It's recommended to use a full path here.
+
+There are a few more if using a Mac M-series machine:
+
+- `MIXED_PRECISION` should be set to `no`.
+- `USE_XFORMERS` should be set to `false`.
+
+#### Dataset considerations
+
+It's crucial to have a substantial dataset to train your model on. There are limitations on the dataset size, and you will need to ensure that your dataset is large enough to train your model effectively. Note that the bare minimum dataset size is `TRAIN_BATCH_SIZE * GRADIENT_ACCUMULATION_STEPS` as well as more than `VAE_BATCH_SIZE`. The dataset will not be useable if it is too small.
+
+Depending on the dataset you have, you will need to set up your dataset directory and dataloader configuration file differently. In this example, we will be using `pseudo-camera-10k` as the dataset.
+
+In your `BASE_DIR` directory, create a multidatabackend.json:
 
 ```json
 [
-    {
-        "id": "pseudo-camera-10k-sd3",
-        "type": "local",
-        "crop": true,
-        "crop_aspect": "square",
-        "crop_style": "center",
-        "resolution": 0.5,
-        "minimum_image_size": 0.25,
-        "maximum_image_size": 1.0,
-        "target_downsample_size": 1.0,
-        "resolution_type": "area",
-        "cache_dir_vae": "cache/vae/sd3/pseudo-camera-10k",
-        "instance_data_dir": "datasets/pseudo-camera-10k",
-        "disabled": false,
-        "skip_file_discovery": "",
-        "caption_strategy": "filename",
-        "metadata_backend": "json",
-    },
-    {
-        "id": "text-embeds",
-        "type": "local",
-        "dataset_type": "text_embeds",
-        "default": true,
-        "cache_dir": "cache/text/sd3/pseudo-camera-10k",
-        "disabled": false,
-        "write_batch_size": 128
-    }
+  {
+    "id": "pseudo-camera-10k-sd3",
+    "type": "local",
+    "crop": true,
+    "crop_aspect": "square",
+    "crop_style": "center",
+    "resolution": 0.5,
+    "minimum_image_size": 0.25,
+    "maximum_image_size": 1.0,
+    "target_downsample_size": 1.0,
+    "resolution_type": "area",
+    "cache_dir_vae": "cache/vae/sd3/pseudo-camera-10k",
+    "instance_data_dir": "datasets/pseudo-camera-10k",
+    "disabled": false,
+    "skip_file_discovery": "",
+    "caption_strategy": "filename",
+    "metadata_backend": "json"
+  },
+  {
+    "id": "text-embeds",
+    "type": "local",
+    "dataset_type": "text_embeds",
+    "default": true,
+    "cache_dir": "cache/text/sd3/pseudo-camera-10k",
+    "disabled": false,
+    "write_batch_size": 128
+  }
 ]
 ```
 
-#### Download dataset
+The, navigate to the `BASE_DIR` directory and create a `datasets` directory:
 
+```bash
 Execute the following commands:
 
 ```bash
```

Comment:
i think this might begin the block twice?

Reply:
![image](https://github.com/bghira/SimpleTuner/assets/59658056/c0c4529e-3120-4ecc-bb5d-6a8283b995bf)

Reply:
Then, *

as well

Reply:
Ahh, how did I miss that, thanks!